pg/scripts/init.sql

```diff
@@ -52,61 +49,62 @@ create table reactions (
     sabio_rk text
 );
 
+create table organisms (
+    kegg text primary key not null,
+    entry text,
+    name text,
+    taxa text
+);
+
+create table domains (
+    abbreviation text primary key not null,
+    name text
+);
 
+create table genes (
+    kegg text,
+    ncbigene text,
+    uniprot text
+);
 
 -- import data
 copy enzymes from program 'cat /input_data/enzymes/*.txt' delimiter E'\t';
 
 copy compounds from '/input_data/supplementary/compound.txt' delimiter E'\t' CSV HEADER;
 copy ec from '/input_data/supplementary/ec.txt' quote E'\u0007' delimiter E'\t' CSV HEADER;
 copy reactions from '/input_data/supplementary/reaction.txt' delimiter E'\t' CSV HEADER;
-
-
+copy organisms from '/input_data/supplementary/organism.txt' delimiter E'\t' CSV HEADER;
+copy domains from '/input_data/supplementary/domain.txt' delimiter E'\t' CSV HEADER;
+copy genes from program 'cat /input_data/supplementary/gene/*.txt' delimiter E'\t';
 
 -- create lookup indexes
-create index on enzymes("gene");
-create index on enzymes("organism");
-create index on enzymes("domain");
-create index on enzymes("reaction_id");
-create index on enzymes("compound");
+create index on reactions using gist (name, kegg);
+create index on compounds using gist (kegg);
+create index on compounds using gin (string_to_array(name, ';'));
```

Comment:
```suggestion
create index on compounds using gist (name, kegg);
```

Reply:
Addressed in: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1062/commits/8547e6ebf003bd817bf407c1696299f931ade869